crates/debugging/src/contracts_data.rs

```diff
@@ -0,0 +1,67 @@
+use crate::trace::types::{ContractName, Selector};
+use cheatnet::runtime_extensions::forge_runtime_extension::contracts_data::ContractsData as CheatnetContractsData;
+use rayon::iter::IntoParallelRefIterator;
+use rayon::iter::ParallelIterator;
+use starknet::core::types::contract::{AbiEntry, SierraClass};
+use starknet_api::core::{ClassHash, EntryPointSelector};
+use std::collections::HashMap;
+
+/// Data structure containing information about contracts,
+/// including their ABI, names, and selectors that will be used to create a [`Trace`](crate::Trace).
+pub struct ContractsData {
+    abi: HashMap<ClassHash, Vec<AbiEntry>>,
+    contract_name: HashMap<ClassHash, ContractName>,
+    selectors: HashMap<EntryPointSelector, Selector>,
```

Comment:
These names are mix of singular and plural, let's unify it (most likely plural).